packages/backend/src/graphql/resolvers/peer.ts

```diff
@@ -29,10 +29,15 @@ export const getPeers: QueryResolvers<ApolloContext>['peers'] = async (
   )
   return {
     pageInfo,
-    edges: peers.map((peer: Peer) => ({
-      cursor: peer.id,
-      node: peerToGraphql(peer)
-    }))
+    edges: await Promise.all(
+      peers.map(async (peer: Peer) => {
+        const balance = await getBalance(ctx, peer.id)
```

Comment:
This `getBalance` db call could happen up to 100 times (max page size), even when liquidity is not specified in the gql requested (making them all unnecessary). One option is to check info with [graphql-parse-resolve-info](https://www.npmjs.com/package/graphql-parse-resolve-info) first.

Another option is to use a custom resolver for liquidity, which I think solves another issue as well. I elaborate on that here https://github.com/interledger/rafiki/pull/1466/files#r1203146434